test/int/project_test.go

```diff
@@ -47,8 +48,8 @@ var _ = Describe("AtlasProject", func() {
 	AfterEach(func() {
 		if createdProject != nil && createdProject.Status.ID != "" {
 			By("Removing Atlas Project " + createdProject.Status.ID)
-			_, err := atlasClient.Projects.Delete(context.Background(), createdProject.Status.ID)
-			Expect(err).ToNot(HaveOccurred())
+			Expect(k8sClient.Delete(context.Background(), createdProject)).To(Succeed())
+			Eventually(checkAtlasProjectRemoved(createdProject.Status.ID), 600, interval).Should(BeTrue())
```

Comment:
[nit] would suggest to use the smaller timeout - 10 seconds?

Reply:
up - in my understanding in this case when there are no clusters in the project the removal time is expected to be very short or am I missing something?

Reply:
right - that's a copy-paste issue, will fix